From: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Date: Fri, 5 Sep 2025 17:04:24 -0600
Subject: mail command absolute pathname

Bug-Debian: http://bugs.debian.org/588862
Forwarded: not-needed
Last-Update: 2024-07-21

This patch changes the default command to /usr/bin/mail in various
 configuration files and manual pages.
---
 doc/man/man1/strigger.1 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/man/man1/strigger.1 b/doc/man/man1/strigger.1
index dc3104e..fb27f92 100644
--- a/doc/man/man1/strigger.1
+++ b/doc/man/man1/strigger.1
@@ -384,7 +384,7 @@ $ cat /usr/sbin/primary_slurmctld_failure
 strigger \-\-set \-\-primary_slurmctld_failure \\
          \-\-program=/usr/sbin/primary_slurmctld_failure
 # Notify the administrator of the failure using e\-mail
-/bin/mail slurm_admin@site.com \-s Primary_SLURMCTLD_FAILURE
+/usr/bin/mail slurm_admin@site.com \-s Primary_SLURMCTLD_FAILURE
 
 $ strigger \-\-set \-\-primary_slurmctld_failure \\
            \-\-program=/usr/sbin/primary_slurmctld_failure
@@ -403,7 +403,7 @@ $ cat /usr/sbin/slurm_admin_notify
 strigger \-\-set \-\-node \-\-down \\
          \-\-program=/usr/sbin/slurm_admin_notify
 # Notify administrator using by e\-mail
-/bin/mail slurm_admin@site.com \-s NodesDown:$*
+/usr/bin/mail slurm_admin@site.com \-s NodesDown:$*
 
 $ strigger \-\-set \-\-node \-\-down \\
            \-\-program=/usr/sbin/slurm_admin_notify
